### ISSUE TITLE
"Local" rustfmt.toml support

### ISSUE THREAD

**Issue by tp-woven** [bazelbuild/rules_rust#2034]
Hello,

Apologies if this was already discussed, I tried searching for this but couldn't turn anything up.

**TL;DR**: Is it possible to override `rustfmt.toml` for a single package/target?

We are using Bazel in a monorepo, and until now we have been using the default config that comes "bundled" with `rules_rust`. However, for my "project" we want to set some configurations, without requiring _everyone_ in the monorepo to have to conform to them. From looking at the docs (and the imlpementation), it appears that this can only be set at the WORKSPACE level, but maybe I'm missing something? For reference, rustfmt itself [_does_ allow](https://github.com/rust-lang/rustfmt#configuring-rustfmt) local configuration files to override global settings:

> You can create a TOML file called rustfmt.toml or .rustfmt.toml, **place it in the project** or any other parent directory and it will apply the options in that file.
